sql_scripts/3_states.sql

```diff
@@ -1,7 +1,10 @@
 CREATE TABLE IF NOT EXISTS beer.states(
    id   INTEGER NOT NULL PRIMARY KEY,
-   name VARCHAR(50) NOT NULL
+   name VARCHAR(50) NOT NULL,
+   created_at TIMESTAMP without TIME ZONE DEFAULT (now() at time zone 'utc'),
+   updated_at TIMESTAMP without TIME ZONE DEFAULT (now() at time zone 'utc')
 );
+    SELECT beer.trigger_creation('beer','beer.states','tr_b_states');
```

Comment:
The first input parameter, which is "schema_n" already contains the name of the schema, so it should be omitted from "schema_n" variable not to repeat yourself with same schema name in second parameter. Modify beer.trigger_creation function EXECUTE state.